yt_dlp/extractor/tiktok.py

```diff
@@ -431,13 +441,32 @@ def _parse_aweme_video_web(self, aweme_detail, webpage_url, video_id):
 
         for download_url in traverse_obj(video_info, (('downloadAddr', ('download', 'url')), {url_or_none})):
             formats.append({
+                **watermark_info(download_url),
                 'format_id': 'download',
                 'url': self._proto_relative_url(download_url),
                 'ext': 'mp4',
                 'width': width,
                 'height': height,
             })
 
+        for bitrate_info in traverse_obj(video_info, ('bitrateInfo', lambda _, v: v['PlayAddr']['UrlList'])):
+            format_info = traverse_obj(bitrate_info, {
+                'format_id': (('GearName', {str}), ('QualityType', {int}, {str_or_none}), any),
```

Comment:
can we use a portion of the UrlKey as the format_id so that it yields similar format id as before, e.g. bytevc1_540p_xxxxxxx?
we can use the same function `parse_url_key` in `_parse_aweme_video_app` to obtain such id.